Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -63,89 +64,100 @@ universe u v w
 variable {R : Type u} [CommRing R]
 
 /-- The proposition that a sequence indexed by integers is an elliptic sequence. -/
-def isEllSequence (h : ℤ → R) : Prop :=
-  ∀ m n r : ℤ, h (m + n) * h (m - n) * h r ^ 2 =
-    h (m + r) * h (m - r) * h n ^ 2 - h (n + r) * h (n - r) * h m ^ 2
+def EllSequence (W : ℤ → R) : Prop :=
+  ∀ m n r : ℤ, W (m + n) * W (m - n) * W r ^ 2 =
+    W (m + r) * W (m - r) * W n ^ 2 - W (n + r) * W (n - r) * W m ^ 2
```

Comment:
I'd slightly prefer `IsEllSequence`; `EllSequence` could be misunderstood as the type of all elliptic sequences. I don't know if it has become a rule but I'd add `Is` to new Prop-valued defs, even though there does exists some old defs without `Is`.

Reply:
Sounds good!